releasenotes/notes/add-status-render-errors-bf6032eebdc05bad.yaml

```
@@ -0,0 +1,4 @@
+---
+enhancements:
+  - |
+    Add ``Status render errors`` section in ``agent status`` to highlight errors while rendering ``/templates/*.tmpl``.
```

The release notes are aimed to users who aren't familiar to the internal structure of the agent code base. WDYT about:
```suggestion
    The status page now includes a ``Status render errors`` section to highlight errors that occurred while rendering it.
```